ui/src/literature/components/__tests__/__snapshots__/NumberOfResultsWithSelectedItemsNumber.test.jsx.snap

```diff
@@ -1,29 +1,30 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`NumberOfResultsWithSelectedItemsNumber does not render selected when selected is 0 1`] = `
+exports[`NumberOfResultsWithSelectedItemsNumber renders correctly with default props 1`] = `
 <span>
-  <Connect(NumberOfResults) />
+  <Connect(NumberOfResults)
+    namespace="testNamespace"
+  />
 </span>
 `;
 
-exports[`NumberOfResultsWithSelectedItemsNumber renders selected when selected is 1 1`] = `
+exports[`NumberOfResultsWithSelectedItemsNumber renders correctly with numberOfSelected 1`] = `
 <span>
   <FormattedNumber>
-    1
+    5
   </FormattedNumber>
    of
    
-  <Connect(NumberOfResults) />
+  <Connect(NumberOfResults)
+    namespace="testNamespace"
+  />
 </span>
 `;
 
-exports[`NumberOfResultsWithSelectedItemsNumber renders selected when selected is more than 1 1`] = `
+exports[`NumberOfResultsWithSelectedItemsNumber renders correctly with numberOfSelected as 0 1`] = `
```

Comment:
0 or 1?

Reply:
this is autogenerated from snapshot in jest to separate the test cases. No. control over it.